atomic_reactor/inner.py

```diff
@@ -664,7 +663,7 @@ def build_docker_image(self):
                 if not exception_being_handled:
                     raise ex
             finally:
-                self.source.remove_tmpdir()
+                self.source.remove_workdir()  # OSBS2 TBD: Don't remove here, remove in exit task?
```

Comment:
well this is running already after exit plugins

exit task will run only exit plugins, so I am not sure if this comment has much value
since we will have to split runners to tasks anyway

Reply:
That's true, but it runs even if there are no exit plugins.

Example:
```python
class PluginBasedTask:
    def execute(self):
        workflow = DockerBuildWorkflow(...)
        return workflow.build_docker_image()


class PreBuildTask(PluginBasedTask):
    plugins_def = PluginsDef(prebuild=[...])


class PostBuildTask(PluginBasedTask):
    plugins_def = PluginsDef(postbuild=[...])


class ExitTask(PluginBasedTask):
    plugins_def = PluginsDef(exit=[...])
```

If `PluginBasedTask.execute()` is implemented as shown above (as I'm planning to), then all three tasks would remove the source dir at the end. I assume the tasks, or at least workflow execution, will look very different in the future, but for now I think the comment is relevant.